build/launcher-script.sh

```diff
@@ -3,6 +3,45 @@
 # Uncomment for debugging purposes:
 # set -eux
 
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+RESOURCES_DIR="$APP_DIR/resources"
+DESKTOP_DATABASE_DIR="$HOME/.local/share/applications"
+MIME_DATABASE_DIR="$HOME/.local/share/mime"
+CUSTOM_MIME_DIR="$MIME_DATABASE_DIR/packages"
+COZY_NOTE_MIME_TYPE_DECLARATION_FILENAME="vnd.cozy.note+markdown.xml"
+
+function isCozyNoteMimeTypeDeclared() {
+    [[ -f "$CUSTOM_MIME_DIR/$COZY_NOTE_MIME_TYPE_DECLARATION_FILENAME" ]] && return
+
+    false
+}
+
+function declareCozyNoteMimeType() {
+    cp "$RESOURCES_DIR/$COZY_NOTE_MIME_TYPE_DECLARATION_FILENAME" "$CUSTOM_MIME_DIR/$COZY_NOTE_MIME_TYPE_DECLARATION_FILENAME"
+    # rebuild the local MIME database to make sure our new type is known
+    update-mime-database $MIME_DATABASE_DIR
```

Comment:
If the folder `~/.local/share/mime` doesn’t exist, this command will fail. So we could add

    mkdir -p "$CUSTOM_MIME_DIR"

before `update-mime-database`